src/package_test.ts

```diff
@@ -7,4 +7,14 @@ describe('package', () => {
         const v2 = require('../package-lock.json').version;
         expect(v1).to.equal(v2);
     });
+
+    it('package-lock should only reference official public npm registry', () => {
+        const deps = require('../package-lock.json').dependencies;
+        for (const key in deps) {
+            const dep = deps[key];
+            const resolved = new URL(dep.resolved);
+            expect(resolved.hostname).to.equal('registry.npmjs.org');
+            expect(resolved.protocol ).to.equal('https);
```

Comment:
```suggestion
            expect(resolved.protocol).to.equal('https);
```

not sure how this space got in here.

Reply:
fixed.